doc/sphinxext/gen_rst.py

```diff
@@ -671,7 +675,7 @@ def generate_file_rst(fname, target_dir, src_dir, root_dir, plot_gallery):
     time_elapsed = 0
     time_m = 0
     time_s = 0
-    if plot_gallery and fname.startswith('plot'):
+    if plot_gallery and demo_regexp.search(fname):
```

Comment:
You cannot have changes to gen_rst in this PR: they need to go to sphinx-gallery.